website/snippets/_new-sl-setup.md

```diff
@@ -1,31 +1,57 @@
-You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
 
-- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
-    * Enterprise plan &mdash; Developer license with Account Admin permissions. Or Owner with a Developer license, assigned Project Creator, Database Admin, or Admin permissions.
-    * Team plan &mdash; Owner with a Developer license.
-    * If you are using a free trial dbt Cloud account, you are on a trial of the Team plan as an Owner, so you're good to go.
-- You must have a successful run in your new environment.
+Here's how to set up the Semantic Layer in dbt Cloud:
 
-Now that we've created and successfully run a job in your environment, you're ready to configure the semantic layer.
+### 1. Account settings and configuration
+- Navigate to **Account Settings** in the navigation menu.
+- On the **Settings** sidebar, select the specific project you want to enable the Semantic Layer for.
+- In the **Project details** page, navigate to the **Semantic Layer** section. Select **Configure Semantic Layer**  
+<Lightbox src="/img/docs/dbt-cloud/semantic-layer/new-sl-configure.jpg" width="60%" title="Semantic Layer section in the Project Details page"/>
 
-1. Navigate to **Account Settings** in the navigation menu.
-2. Use the sidebar to select your project settings. Select the specific project you want to enable the Semantic Layer for.
-3. In the **Project Details** page, navigate to the **Semantic Layer** section, and select **Configure Semantic Layer**.
+- In the **Semantic Layer & Credentials** page, select the deployment environment you want for the Semantic Layer and click **Save**. This provides administrators with the flexibility to choose the environment where the Semantic Layer will be enabled. <br />
 
-<Lightbox src="/img/docs/dbt-cloud/semantic-layer/new-sl-configure.jpg" width="60%" title="Semantic Layer section in the Project Details page"/>
+### 2. Add Semantic Layer credential and service token
+
+Next, we recommend using role-based access for building credentials based on your teams needs. Assign specific roles to credentials, which can be applied to different users based on team structures. Authenticate to the dbt Semantic Layer in downstream interfaces using a [service token](/docs/dbt-cloud-apis/service-tokens) for more granular control. 
+
```

Comment:
@rpourzand added the physical access clarification - what do you mean by 'physical access' though?

Reply:
The semantic layer has physical access and logical access components. Physical = underlying data which is managed by credentials. This is the only thing we allow users to control today. There is also a component of "logical access" which refers to the metrics and dimensions in the semantic layer (like the virtual objects). This is something that we don't allow admins to control granularly yet. I don't think we need to explicitly call that out, but I think we need to make it clear that these access controls are for physical data, and users can still see all logical objects metrics/dimensions. As an example, I can see there there is a metric revenue, but I might not be able to physically access the underlying data of it when I try.